src/cmd/linuxkit/pkglib/docker.go

```diff
@@ -28,7 +28,7 @@ const (
 )
 
 var platforms = []string{
-	"linux/amd64", "linux/arm64", "linux/s390x",
+	"linux/amd64", "linux/arm64", "linux/s390x", "linux/riscv64",
```

Comment:
I actually don't think this is used anywhere. I cannot find it referenced, and when I commented it out and ran `go vet`, I had no complaints. Maybe it should just be removed?

Reply:
Yes, can you remove it, and see that everything works?

Reply:
It does! Thanks for the hint @deitch